server/src/routes/api/v1/recipes/recipe.route.ts

```diff
@@ -17,14 +17,14 @@ export class RecipeRoute {
     this._router.post("/",
       this._authValidationHandler.checkToken,
       this._authValidationHandler.checkUser,
-      this._authValidationHandler.nonRestricted,
+      //this._authValidationHandler.nonRestricted,
       this._modelValidationHandler.validate,
       this._recipeHandler.save);
 
     this._router.get("/:id",
       this._authValidationHandler.checkToken,
-      this._authValidationHandler.checkUser,
```

Comment:
If you do this, than is the decoded object still inside the request body. This is the reason why I have added the che kUser middleware.